benchmark/Benchmark/Main.purs

```diff
@@ -2,134 +2,141 @@ module Benchmark.Main (main) where
 
 import Prelude
 
-import Control.Monad.Eff
-import Control.Monad.Free
-import Control.Monad.Trampoline
-
-import Data.Array
-import Data.Foldable
-
-import Benchotron.Core
-import Benchotron.UI.Console
-
+import Benchmark.Free326759a as Free326759a
+import Benchmark.Trampoline0df59c5 as Trampoline0df59c5
+import Benchmark.Trampoline326759a as Trampoline326759a
```

Comment:
I found it easier to tell what code came from what module by qualifying both of them with their commit, instead of only qualifying the old v0.6.1 code.